pulsar-common/src/main/proto/PulsarMarkers.proto

```diff
@@ -32,6 +32,9 @@ enum MarkerType {
     REPLICATED_SUBSCRIPTION_UPDATE            = 13;
 
     // Next markers start at 20
+    TXN_DATA = 20;
```

Comment:
I don't think we need TXN_DATA. TXN_DATA are just normal messages, right?

Reply:
Ah. Yes. I will remove it.